app/Console/Kernel.php

```diff
@@ -59,6 +59,10 @@ protected function schedule(Schedule $schedule)
             ->hourlyAt(15)
             ->graceTimeInMinutes(15);
 
+        $schedule->command('waitinglists:processpendingremovals')
```

Comment:
I don't think this interval is going to work. There is no assurance on the processing time of incoming network data from the datafeed. I think once daily is the best assurance you will get.